Scenarioo/ScenariooTest/ScenariooTest/ScenarioDocuWriterTest.cs

```diff
@@ -283,30 +356,31 @@ public void WriteTreeStructureInDetails()
 
             scenario.AddDetail("exampleTree", rootNode);
 
-            // WHEN: the object was saved.
-            this.writer.SaveScenario(UseCaseName, scenario);
+            // act
+            writer.SaveScenario(SerializationUseCase, scenario);
+            writer.Flush();
 
-            // THEN: the files are not created directly but asynchronously. Flush will wait until all Tasks are finished.
-            this.writer.Flush();
+            // assert
+            Assert.IsTrue(File.Exists(docuFiles.GetScenarioFile(BranchName, BuildName, SerializationUseCase, ScenarioName)));
 
-            // THEN: the scneario file exists
-            Assert.IsTrue(File.Exists(this.docuFiles.GetScenarioFile(BranchName, BuildName, UseCaseName, ScenarioName)));
+            scenario.Status = "success";
+            writer.SaveScenario(SerializationUseCase, scenario);
         }
 
-        [TestMethod]
+        [Test] // TODO: review this async stuff, i'm not sure about that!
```

Comment:
@tobiaszuercher please introduce issues for things like this on the scenarioo-cs repo, and mention in TODO comment. Thank you!